package.json

```diff
@@ -51,8 +51,9 @@
     "@typescript-eslint/parser": "^6.4.1",
     "browser-sync": "^2.29.3",
     "concurrently": "^8.2.0",
-    "eslint": "^8.47.0",
+    "eslint": "^8.48.0",
     "eslint-config-prettier": "^9.0.0",
+    "eslint-import-resolver-typescript": "^3.6.0",
```

Comment:
This resolver [is needed](https://github.com/import-js/eslint-plugin-import#typescript) to properly support resolving TypsScript files.